packages/fast-components-react-base/src/index.spec.ts

```diff
@@ -0,0 +1,35 @@
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure } from "enzyme";
+import * as fs from "fs";
+import * as path from "path";
+
+configure({adapter: new Adapter()});
+
+describe("index.ts component exports", (): void => {
+    test("All components have respective class exports in index.ts", () => {
+        const isDirectory: any = (source: any): any => fs.lstatSync(source).isDirectory();
+        const getDirectories: any = (source: any): any =>
+            fs.readdirSync(source).map((name: any) => path.join(source, name)).filter(isDirectory);
+        const srcDir: string[] = getDirectories("src");
+        const components: string[] = [];
+        const index: any = path.resolve(__dirname, "index.ts");
+
+        srcDir.forEach((entry: string) => {
+            components.push(entry.slice(4).replace(/-/g, ""));
+        });
+
+        const data: any = fs.readFileSync(index, "UTF-8");
+        const found: any = [];
+        const rxp: RegExp = /{([^}]+)}/g;
+        let curMatch: any;
```

Comment:
Can we clean up some of these "any" type defs? For instance, line 22 is clearly an array unless you're  changing types...to which I would ask, "why"?

Since we have "typings" in here, can you update typings that are actually known? For those that are truly *unknown* (which I'm not sure there are any) we should use the *unknown* typedef.

Reply:
Will all be updated in new PR